frontend/src/metabase/components/Toggle.jsx

```diff
@@ -23,6 +23,7 @@ export default class Toggle extends Component {
     const { value, small, className, color, onChange, ...props } = this.props;
     return (
       <a
+        data-testid="toggle"
```

Comment:
If we can't avoid `data-testid` here, how about making it much more specific than "toggle" to avoid ambiguities in the future?

Reply:
Good call out. The primary issue here is that we frequently rely on section headers to label inputs which means we can't use `getByLabelText` to target the input. This suggests a gap in accessibility. I've replaced the `data-testid` attribute with an `aria-label` attribute which lets me use `getByLabelText` and also fixes this accessibility problem. Thanks!

Reply:
Nice solution 👍